src/utils/classifyIPAddress.js

```diff
@@ -183,33 +183,36 @@ const ipv6Ranges = [
 	}
 ]
 
-
 function classifyIPv4(ipAddress) {
+	const ip = BigInt(ipAddress); // Ensure input is BigInt
```

Comment:
We are losing precision here already, it must be fixed "at the sorcre" (conversion of string to BigInt)